components/CryptoItem.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import {Text} from 'react-native';
+import MaterialIcons from 'react-native-vector-icons/MaterialIcons';
+import styled from 'styled-components/native';
+
+type CryptoItemProps = {
+  id: string;
+  name: string;
+  symbol: string;
+  price: number;
+  percentageChange: number;
+};
+
+const Container = styled.View`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px 30px;
+  border-bottom-width: 1px;
+  border-color: #ccc;
```

Comment:
use Theme, styled components allow the direct use of it: https://styled-components.com/docs/advanced